include/pistache/http.h

```diff
@@ -549,7 +547,7 @@ class ResponseWriter : public Response {
        return &buf_;
     }
 
-    DynamicStreamBuf *rdbuf([[maybe_unused]] DynamicStreamBuf* other) {
```

Comment:
Why are you removing the "[[maybe_unused]]"?  It was added only 28 days ago with PR #259.

Reply:
I got warning for this. And then I read this comment also added in that PR:
https://github.com/oktal/pistache/pull/259/files#diff-5a4ac07535107245765a4a8d3f822a83R52

Reply:
I do not want to require c++17 any time soon.  So for the sake of consistency, removing this is fine.  Please find the other references are remove them as well (there is only one other, in that comment).